### DISCUSSION TITLE
How to package Theia Plugin

### DISCUSSION THREAD

**Discussion by taiyivictor** [eclipse-theia/theia#8040] [General] [answered]
How can I package Theia Plugin? 
Thanks.

**Comment by akosyakov** [comment]
Please see https://theia-ide.org/docs/authoring_plugins/

**Reply by taiyivictor** [reply]
When I develop a plugin refer to the link and want to send to other person for use. I think we should package it firstly, for the developed plugins takes tens of MBs, attached link seems no package and publish method.

**Reply by akosyakov** [reply]
cc @benoitf

**Comment by zhaomenghuan** [comment, marked as answer]
You can use the [vsce](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) module to package your plugin into .vsix file，and use the [ovsx](https://www.npmjs.com/package/ovsx) module to publish to open-vsx.org.

Please see https://github.com/eclipse/openvsx/wiki/Publishing-Extensions
